Library/Formula/zeromq.rb

```diff
@@ -28,6 +28,12 @@ class Zeromq < Formula
   depends_on "libpgm" => :optional
   depends_on "libsodium" => :optional
 
+  if not build.head?
+    # enable --without-libsodium on libzmq < 4.1
+    # zeromq/zeromq4-x#105
+    patch :DATA
```

Comment:
The upstream patch won't be what's applied here, since only `configure.ac` is tracked in the upstream repo. So here I have only the result of the patch on configure. Should I make a gist for this patch, then? Alternately, I could pull in the upstream patch on `configure.ac` and re-run `sh autogen.sh`.